### PR TITLE
Resolve statics for publish diff revisions

### PR DESCRIPTION
**Pull request by MiroDojkic**
### This PR
Fixes diff preview for `tce-video` and other elements that require asset resolving

#### 🔍 QA note
- this PR affects history/revisions as well

### CODE REVIEW

**Comment by underscope** [comment]
@kjuej @markolovric

**Comment by markolovric** [comment]
I can preview newly added and uploaded audio & video elements, but cant preview previously added and uploaded after publishing.

image

File: server/revision/revision.service.js [hunk 1]
```diff
@@ -14,44 +15,62 @@ async function getEntityRemovesSinceMoment(activity, timestamp) {
...
-        [Op.or]: [{
-          id: { [Op.in]: ids }
-        }, {
-          activityId: { [Op.in]: activityIds }
-        }]
-      }
-    }
-  });
+function resolveStaticsForEach(revisions) {
+  return Promise.all(revisions.map(async revision => {
```
Comment (underscope) [inline server/revision/revision.service.js]: Promise.map 🤔 ?

File: server/revision/revision.service.js [hunk 2]
```diff
@@ -14,44 +15,62 @@ async function getEntityRemovesSinceMoment(activity, timestamp) {
...
-      state: {
-        [Op.or]: [{
-          id: { [Op.in]: ids }
-        }, {
-          activityId: { [Op.in]: activityIds }
-        }]
-      }
-    }
-  });
+function resolveStaticsForEach(revisions) {
```
Comment (underscope) [inline server/revision/revision.service.js]: Resolve in hook? (Controlled by context flag) ?

**Pull request merged**